.github/workflows/cifuzz.yml

```diff
@@ -1,12 +1,5 @@
 name: CI Fuzz
-on:
-  pull_request:
-    paths:
-      - 'include/**'
-      - 'src/**'
-      - 'tests/oss-fuzz/**'
-    types:
-      - closed
+on: [pull_request]
```

Comment:
The syntax looks quite different from the original code. I am not familiar with GitHub workflow's YML files. Might want to have someone else check this change.

Reply:
Also the changes in this file seem unrelated to the fuzztest targets. If so, it would be good to describe these changes in the commit message (or move them to a separate pull request).

Reply:
I just realized this pull request was a pure revert of commit a12e0dc6c8a72959603797875163bcb509f98d23.